### PR TITLE
Update some AIP RFC links to later commits to capture clarifications and RFC status changes

### PR DESCRIPTION
**Pull request by swcurran**
Added a changelog to AIP 2.0 to document the reasons for the link updates (changing the commit).

### CODE REVIEW

File: concepts/0302-aries-interop-profile/README.md [hunk 1]
```diff
@@ -229,6 +229,19 @@ The important feature that Aries implementers should understand to differentiate
 agent can only use the RFC 0019-encryption-envelope present. If it is present, the values indicate the envelope format(s)
 the agent does support. See the RFCs for additional details.
 
+#### Changelog - AIP 2.0
+
+The original commit used in the definition of AIP 2.0 was: b3a3942ef052039e73cd23d847f42947f8287da2
+
+The following clarifications have been made to RFCs that make up AIP 2.0:
+
+- RFC 0003 Protocols: A correction to the tic-tac-toe example protocol was made.
```
Comment (TimoGlastra) [inline concepts/0302-aries-interop-profile/README.md:238]: Update to 0646 is missing in commit updates and changelog

File: concepts/0302-aries-interop-profile/README.md [hunk 2]
```diff
@@ -229,6 +229,19 @@ The important feature that Aries implementers should understand to differentiate
...
+- RFC 0017 Attachments: A clarification was made around the use of base64url encoding/decoding.
+- RFC 0434 Out of Band: The RFC status was changed to "Accepted."
+- RFC 0627 Static Peer DIDs: The RFC status was changed to "Accepted."
+- RFC 0441 Present Proof Best Practices: A convention for representing dates to enable simple "older than" predicates was added.
+- RFC 0501 DIF Presentation Exchange Attachment: The RFC Status was changed to "Accepted."
```
Comment (TimoGlastra) [inline concepts/0302-aries-interop-profile/README.md:243]:
```suggestion
- RFC 0510 DIF Presentation Exchange Attachment: The RFC Status was changed to "Accepted."
```

Comment (swcurran) [reply concepts/0302-aries-interop-profile/README.md:243]: Doh...

**Review by TimoGlastra — APPROVED** [review]
Awesome

**Review by TelegramSam — APPROVED** [review]

**Pull request merged**